src/laser/cooling.rs

```diff
@@ -26,6 +27,16 @@ pub struct CoolingLight {
 	/// wavelength of the laser light, in SI units of m.
 	pub wavelength: f64,
 }
+
+impl Lerp<CoolingLight> for CoolingLight {
```

Comment:
I think this seems like a sensible implementation to add to base. 

However, this could also use the auto-implementation for Lerp defined in the atomecs macros crate. It seems essentially the same. Or is the issue that the integer polarization cannot be lerped? If so, I can change the auto-implementation to provide lerp support for integers (and that will fix more widely).

Reply:
yes! The integer polarization is indeed at fault here. 

What would the auto-implementation look like? Just ignore integers? Or make them float? (seems dangerous)

Reply:
probably just round them